lib/snapshot/mem_fs.go

```diff
@@ -710,3 +711,79 @@ func (fs *MemFS) untarFile(path string, header *tar.Header, r *tar.Reader) error
 	}
 	return nil
 }
+
+// Public api for diff image purpose.
+func CompareFS(fs1, fs2 *MemFS, image1Name, image2Name image.Name, ignoreModTime bool) {
```

Comment:
// CompareFS is the public API for comparing merged layers of two images for differences.